connectors/presto/readme.md

```diff
@@ -1 +1,111 @@
 # IBM created Presto connectors that may be useful to you
+
+## Snowflake connector
+
+### Overview
+
+Snowflake is a cloud-based data platform that allows users to store, analyze, and share data.
+
+The Snowflake connector allows querying and creating tables in an external
+Snowflake database. This can be used to join data between different
+systems like Snowflake and Hive, or between two different Snowflake accounts.
+
+### Connector Configuration
+
+To configure the Snowflake connector, create a catalog properties file in *etc/catalog* named, for example, *snowflake.properties*, to mount the Snowflake connector as the snowflake catalog. Create the file with the following contents, replacing the connection properties as appropriate for your setup:
+
+```
+connector.name=snowflake
+connection-url=jdbc:snowflake://<account_identifier>.snowflakecomputing.com/?db={dbname}  
+connection-user={username}
+connection-password={password}
+```
+
+The `connection-url` defines the connection information and parameters to pass to the Snowflake JDBC driver.
+
+Syntax: `jdbc:snowflake://<account_identifier>.snowflakecomputing.com/?<connection_params>`
+
+User has to provide only the `account_identifier`, and shouldn't include the prefix `.snowflakecomputing.com`
+
+The `connection-user` and `connection-password` are typically required and determine the user credentials for the connection.
+
+### SSL Configuration
+
+By default, connections to Snowflake use SSL.
+
+### Integrating the Snowflake Plugin to Presto   
+
+1. Untar *watsonx-data/connectors/presto/snowflake_tarball/presto-snowflake-0.288/snowflake.tar* and place the plugin in the production’s plugins directory(presto-server-<version>/plugin).
```

Comment:
Should we instead include instructions to build this project, and retrieve it from the project's target directory?  This tar will become out of date immediately after the next PR is merged.